adcm_aio_client/core/objects/cm.py

```diff
@@ -36,7 +38,50 @@ def get_own_path(self: Self) -> Endpoint:
         return ("adcm",)
 
 
-class Bundle(Deletable, InteractiveObject): ...
+class License(InteractiveObject): ...
+
+
+class Bundle(Deletable, RootInteractiveObject):
+    PATH_PREFIX = "bundles"
+
+    @property
+    def name(self: Self) -> str:
+        return str(self._data["name"])
+
+    @property
+    def display_name(self: Self) -> str:
+        return str(self._data["display_name"])
+
+    @property
+    def version(self: Self) -> str:
+        return str(self._data["version"])
+
+    @property
+    def edition(self: Self) -> Literal["community", "enterprise"]:
+        return self._data["edition"]
+
+    @property
+    def signature_status(self: Self) -> Literal["invalid", "valid", "absent"]:
+        return self._data["signatureStatus"]
+
+    @property
+    def _type(self: Self) -> Literal["cluster", "provider"]:
+        return self._data["mainPrototype"]["type"]
+
+    @CachedProperty
+    async def license(self: Self) -> License:
+        response = await self._requester.get(*self.get_own_path())
+        return self._construct(what=License, from_data=response.as_dict()["mainPrototype"]["license"])
+
+    def get_own_path(self: Self) -> Endpoint:
+        return self.PATH_PREFIX, self.id
+
+
+class BundlesNode(PaginatedAccessor):
+    class_type = Bundle
+
+    def get_own_path(self: Self) -> Endpoint:
+        return ("bundles",)
```

Comment:
I believe most other places use passing path during construction `BundlesNode(path=("bundles", ), requester=self._requester)`

let's use that approach instead of overriding `get_own_path` for a time being

Reply:
removed from here

Reply:
it's still `return BundlesNode(path=(), requester=self._requester)`